lib/host/wasmedge_process/processfunc.cpp

```diff
@@ -283,17 +291,20 @@ WasmEdgeProcessRun::body(Runtime::Instance::MemoryInstance *MemInst) {
   Env.StdIn.clear();
   Env.TimeOut = Env.DEFAULT_TIMEOUT;
   return Env.ExitCode;
-};
+#elif WASMEDGE_OS_WINDOWS
+  return Unexpect(ErrCode::ExecutionFailed);
```

Comment:
Should we have another error message to tell that `process module` is disable on Windows platform?